modules/virt-booting-efi-mode.adoc

```diff
@@ -5,21 +5,21 @@
 [id="virt-booting-vms-efi-mode_{context}"]
```

Comment:
From the [OCP docs contribution guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata): The module ID must match the module's file name.